Add joomla and murmurhash files (#170)
odidev/ecosystem-dashboard-for-arm: content/opensource_packages/joomla.md

---
name: Joomla
category: Content mgmt platforms

> description: Joomla is an content management system (CMS), which enables you to build web sites and powerful online applications.

Author: ```suggestion
description: Joomla is a content management system (CMS), which enables the user to build web sites and powerful online applications.
```

> download_url: https://downloads.joomla.org/

Author: I can't open this link. Kindly review again.

works_on_arm: true
supported_minimum_version:
    version_number: 3.8.10
    release_date: 26/06/2018

optional_info:
    homepage_url: https://www.joomla.org
    support_caveats:
    alternative_options:
    getting_started_resources:
        arm_content:
        partner_content:
        official_docs: https://docs.joomla.org/J3.x:Getting_Started_with_Joomla!
    arm_recommended_minimum_version:
        version_number:
        release_date:

optional_hidden_info:
    release_notes__supported_minimum:
    release_notes__recommended_minimum:

>     other_info: Arm64 support is  not mentioned in the release note although it can be found at dockerhub release: https://hub.docker.com/_/joomla/tags?page=59&page_size=&name=&ordering=

Author: ```suggestion
    other_info: Linux/ARM64 support is not mentioned in the release notes, although it can be found at the DockerHub releases [here](https://hub.docker.com/_/joomla/tags?page=59&page_size=&name=&ordering=).
```
